azure-pipelines.yml

```diff
@@ -38,6 +38,12 @@ steps:
     set -x
     docker run --name test-run-image -d $(IMAGE_NAME)
     docker exec test-run-image rake "parallel:spec[,, -O .azure_parallel]"
```

Comment:
Should this be running default `rake` to include linting and brakeman?

Reply:
Not sure tbh. There are separate tasks for linting and brakeman. @TobyRet setup the parallel testing for us so might be able to comment

Reply:
If they're run elsewhere then that's fine. Just wasn't sure 👍